Document/0x06b-Basic-Security-Testing.md

```diff
@@ -359,7 +351,29 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
 
 The following sections describes on how to retrieve basic information of an iOS app, that might be useful during a penetration test.
 
--- ToDo Objection: https://github.com/OWASP/owasp-mstg/issues/1247
+Let's focus first on objection. In the following example Frida is running as server on a jailbroken device and the app iGoat is running in the foreground. To attach to a process in this scenario you need to use the flag `--gadget` with the process name. When using objection you can retrieve different kind of information, where `env` will show you all the directory information of iGoat.
+
+```bash
+$ objection  --gadget "iGoat-Swift" explore
+...
+OWASP.iGoat-Swift on (iPhone: 10.3.3) [usb] # env
```

Comment:
maybe you can redo it on my ios 12 device :D ? stuff changed quiet a lot. So maybe we can have it both for ios 10 and 12.. Same holds for the other hacks...

Reply:
OVerall: please check most of the ios stuff and put the 12 next to it :) ?

Reply:
JeroenB will explain frida-ios-dump instead of clutch to address the iOS 12 issue. 

The directory structure was now still the same on iOS12:

```
objection --gadget MSTG-JWT explore
Using USB device `iPhone`
Agent injected and responds ok!

     _     _         _   _
 ___| |_  |_|___ ___| |_|_|___ ___
| . | . | | | -_|  _|  _| | . |   |
|___|___|_| |___|___|_| |_|___|_|_|
        |___|(object)inject(ion) v1.6.4

     Runtime Mobile Exploration
        by: @leonjza from @sensepost

[tab] for command suggestions
org.owasp.MSTG.training on (iPhone: 12.0) [usb] # env

Name               Path
-----------------  -------------------------------------------------------------------------------------------
BundlePath         /var/containers/Bundle/Application/D00BFAD6-A2C0-4DBB-B3B7-D153CA08468E/MSTG-JWT.app
CachesDirectory    /var/mobile/Containers/Data/Application/8E4A50FC-4C3D-4E7F-89B4-CA5E1DFBA820/Library/Caches
DocumentDirectory  /var/mobile/Containers/Data/Application/8E4A50FC-4C3D-4E7F-89B4-CA5E1DFBA820/Documents
LibraryDirectory   /var/mobile/Containers/Data/Application/8E4A50FC-4C3D-4E7F-89B4-CA5E1DFBA820/Library
```